packages/client/components/MyDashboardTimeline.tsx

```diff
@@ -47,6 +47,7 @@ const MyDashboardTimeline = (props: Props) => {
         $userIds: [ID!]
         $eventTypes: [TimelineEventEnum!]
         $teamIds: [ID!]
+        $includeArchived: Boolean = true
```

Comment:
+1 Instead of setting a default here, I would prefer to just have the variable defined and then we pass the value explicitly. This way I only have to check in one place and not multiple what this value will be like.